deepmd/pt/model/descriptor/dpa1.py

```diff
@@ -108,11 +108,17 @@ def get_dim_out(self) -> int:
     def get_dim_emb(self) -> int:
         return self.se_atten.dim_emb
 
-    def distinguish_types(self) -> bool:
-        """Returns if the descriptor requires a neighbor list that distinguish different
-        atomic types or not.
+    def mixed_types(self) -> bool:
+        """If true, the discriptor
+        1. assumes total numbe of atoms aligned across frames;
+        2. requires a neighbor list that does not distinguish different atomic types.
+
+        If false, the discriptor
+        1. assumes total numbe of atoms of each atom type aligned across frames;
+        2. requires a neighbor list that distinguishes different atomic types.
```

Comment:
TF requires both central and neighbor atoms to be sorted by atom types. Does PT have this requirement?

Reply:
pt does not support "type two side", i.e. it only use different nets for different neighbor types, but not for different center atom types, thus the center atoms are not assumed to be sorted.